examples/pass-server-data/routes/item.js

```diff
@@ -0,0 +1,16 @@
+const api = require('../operations/get-item')
+
+// Returns the express handler function. Necessary so that we can get the nextApp instance.
+function getItemRenderer (nextApp) {
+  return (req, res) => {
+    console.log('Express is handling request')
+    const itemData = api.getItem()
+
+    res.format({
```

Comment:
The res.format is to support isomorphic rendering of that page so it works when linked to. I could also mount 2 separate routes, one at `/item` which renders the html, another at maybe `/_data/item` which sends the json. But I've found that doing it in the same route method is more concise.

I can move the route to server.js though, and leave how they break things off up to the reader.